engine/CMakeLists.txt

```
@@ -169,6 +169,8 @@ set(CUBOS_ENGINE_SOURCE
 	"src/render/shadow_atlas_rasterizer/shadow_atlas_rasterizer.cpp"
 	"src/render/cascaded_shadow_maps/plugin.cpp"
 	"src/render/cascaded_shadow_maps_rasterizer/plugin.cpp"
+	"src/render/active_component/plugin.cpp"
+	"src/render/active_component/active_component.cpp"
```

```suggestion
	"src/render/active/plugin.cpp"
	"src/render/active/active.cpp"
```
You should rename these files to this, which is more in line with the rest of the code base.
Don't forget to do the same for the include files!